uat/steps/component.py

```diff
@@ -0,0 +1,69 @@
+import os
+import t_utils
+
+from behave import step
+from pathlib import Path
+from constants import (
+    GG_CONFIG_JSON, GG_RECIPE_YAML, GG_BUILD_DIR, GG_BUILD_ZIP_DIR,
+    DEFAULT_AWS_REGION, DEFAULT_S3_BUCKET_PREFIX, DEFAULT_ARTIFACT_AUTHOR
+)
+
+
+@step('we verify component files')
+def verify_component_files(context):
+    cwd = context.cwd if "cwd" in context else os.getcwd()
+    assert Path(cwd).joinpath(GG_RECIPE_YAML).resolve().exists(), f"{GG_RECIPE_YAML} does not exist"
+    assert Path(cwd).joinpath(GG_CONFIG_JSON).resolve().exists(), f"{GG_CONFIG_JSON} does not exist"
+
+
+@step('we verify component zip build files')
+def verify_component_zip_build_files(context):
+    cwd = context.cwd if "cwd" in context else os.getcwd()
+    assert Path(cwd).joinpath(GG_BUILD_DIR).resolve().exists()
+    assert Path(cwd).joinpath(GG_BUILD_ZIP_DIR).resolve().exists()
+
+
+@step('we verify component build files')
+def verify_component_build_files(context):
+    cwd = context.cwd if "cwd" in context else os.getcwd()
+    assert Path(cwd).joinpath(GG_BUILD_DIR).resolve().exists()
+
+
+@step('we verify build artifact {artifact_name}')
```

Comment:
can we change it to `we verify build artifact named {artifact_name}`

Reply:
done